cve_bin_tool/version_compare.py

```diff
@@ -140,10 +140,14 @@ def version_compare(v1: str, v2: str):
                 # Honestly it's hard to guess if .dev3 is going to be more or less than .rc4
                 # unless you know the project, so hopefully people don't expect that kind of range
                 # matching
```

Comment:
Maybe just me having a hard time with regular expressions, but the simplest way to follow the code is by internalizing the first comment that you have here ("They're both of type...") while the second one ("Honestly it's hard...") distracts me. I know this is nitpicking, but I with switch the order of those two comments and leave the one relative to the code immediately before the code.

Reply:
@antoniogi I *think* it's more clear if I explain what happens, so I changed this comment.  Do you mind re-reading it and seeing if it helps make things more clear?